hack/lib/golang.sh

```diff
@@ -214,10 +214,9 @@ kubeedge::golang::build_binaries() {
     echo "building $bin"
     local name="${bin##*/}"
     set -x
-    go build -o ${KUBEEDGE_OUTPUT_BINPATH}/${name} -gcflags="${gogcflags:-}" -ldflags "${goldflags:-}" $bin
+    GOOS="linux" CGO_ENABLED=1 go build -o ${KUBEEDGE_OUTPUT_BINPATH}/${name} -gcflags="${gogcflags:-}" -ldflags "${goldflags:-}" $bin
```

Comment:
for mac os, must this `CGO_ENABLED=1` be required?

Reply:
Not needed for macOS, I saw that some of the binaries needed it. I can remove `CGO_ENABLED=1` if they don't all need CGO

Reply:
I think now we have no conclusion that kubeedge only supports Linux, so if we add `GOOS="linux"` here, all the other os   need to be tested.